app/uk/gov/hmrc/helptosavetestadminfrontend/controllers/HelpToSaveApiController.scala

```diff
@@ -225,4 +175,36 @@ class HelpToSaveApiController @Inject()(http: WSHttp, authConnector: AuthConnect
     )
   }
 
+  private def toCurlRequestLines(httpHeaders: HttpHeaders): String =
+    httpHeaders.toMap().map{ case (k,v) ⇒ s"""-H "$k: $v" \\"""}.mkString("\n")
```

Comment:
Do we need a $ in front of the k?

Reply:
yes, to get the actual underlying string behind `k` rather than having the character `k` in the resulting string

Reply:
Ah I see